corehq/apps/sms/tests/test_backends.py

```diff
@@ -37,127 +39,145 @@
 from urllib import urlencode
 
 
-class AllBackendTest(BaseSMSTest):
+class AllBackendTest(DomainSubscriptionMixin, TestCase):
 
-    def setUp(self):
-        super(AllBackendTest, self).setUp()
-
-        self.domain_obj = Domain(name='all-backend-test')
-        self.domain_obj.save()
-        self.create_account_and_subscription(self.domain_obj.name)
-        self.domain_obj = Domain.get(self.domain_obj.get_id)
+    @classmethod
+    def setUpClass(cls):
```

Comment:
needs to call super's setupclass. same for teardownclass. there's a warning right after this box: https://docs.djangoproject.com/en/1.10/topics/testing/tools/#django.test.SimpleTestCase.allow_database_queries